Kernel/Memory/MemoryManager.cpp

```diff
@@ -1071,7 +1071,7 @@ u8* MemoryManager::quickmap_page(PhysicalAddress const& physical_address)
 {
     VERIFY_INTERRUPTS_DISABLED();
     auto& mm_data = get_data();
-    mm_data.m_quickmap_prev_flags = mm_data.m_quickmap_in_use.lock();
+    mm_data.m_quickmap_interrupts_state_before_locking = mm_data.m_quickmap_in_use.lock();
```

Comment:
This code looks kinda silly now because it saves the previous state in mm_data, but 2 lines up we VERIFY that interrupts are currently disabled 🤔

Reply:
Hmm yeah, so that means we actually don't need to store the `previous_interrupts_state`. I could remove the field from `mm_data`, but if in the future we might remove the `VERIFY_INTERRUPTS_DISABLED` in this function, then we would need to add it again. What do you think is better?